roles/requirements.yml

```diff
@@ -1,3 +1,4 @@
 ---
 roles:
-  - name: hurricanehrndz.rustup
+  - src: https://github.com/kowshikvajrala/ansible-rustup.git
```

Comment:
I have forked your repo, could you please change it to encapsulate https://github.com/encapsulate-xyz/ansible-rustup